### DISCUSSION TITLE
Quota 100 pesan per hari

### DISCUSSION THREAD

**Discussion by budiaman** [kirimwa-id/forum#148] [Tanya Jawab] [answered]
Selamat pagi mas Rio.
Untuk terkait limit quota 100 pesan per hari yang di infokan di email saat pendaftaran token.
Limit ini untuk incoming message yang masuk atau limit yang outgoing message ya mas?

**Comment by kafaa-dev** [comment, marked as answer]
Sepertinya limit untuk outgoing message (pengiriman pesan)


> Tidak ada batas untuk banyaknya pesan masuk atau incoming message.

https://developer.kirimwa.id/api/v1/#quotas

**Reply by budiaman** [reply]
Baik, terima kasih atas informasinya

**Reply by kafaa-dev** [reply]
Sama sama mas ...
